src/features/events/models/EventDataModel.ts

```diff
@@ -1,6 +1,7 @@
 import Environment from 'core/env/Environment';
 import { IFuture } from 'core/caching/futures';
 import { ModelBase } from 'core/models';
+import theme from 'theme';
```

Comment:
It's not ideal to use the theme like this, or to use it inside a model at all. It will cause problems in case we make themes dynamic (which we will likely want to do to implement "night mode" at some point). But let's ignore that for now because we will likely abandon models before we implement night mode anyway.